code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -2368,6 +2368,121 @@
 	
 	if(prob(5))
 		M.emote(pick("twitch","blink_r","shiver"))
+		
+/datum/reagent/hypozine //syndie hyperzine
+	name = "Hypozine"
+	id = HYPOZINE
+	description = "Hypozine is an extremely effective, short lasting, muscle stimulant."
+	reagent_state = LIQUID
+	color = "#C8A5DC" //rgb: 200, 165, 220
+	var/has_been_hypozined = 0
+	var/has_had_heart_explode = 0 //We've applied permanent damage.
+	custom_metabolism = 0.04
+	var/oldspeed = 0
+	data = 1
+
+/datum/reagent/hypozine/reagent_deleted()
+
+	if(..())
+		return 1
+
+	if(!holder)
+		return
+	var/mob/M =  holder.my_atom
+
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		if(!has_been_hypozined || has_had_heart_explode)
+			return
+		var/timedmg = ((120 SECONDS) - data) / 10
+		dehypozine(H, timedmg * 3, 1, 0)
+
+/datum/reagent/hypozine/on_mob_life(var/mob/living/M)
+
+	if(..())
+		return 1
+	
+	M.reagents.add_reagent ("hyperzine", 0.03) //To pretend it's all okay.
+	if(ishuman(M))
+		if(data<121 && !has_been_hypozined)
+			has_been_hypozined = 1
+			has_had_heart_explode = 0 //Fuck them UP after they're done going fast.
+			oldspeed = M.movement_speed_modifier
+			
+	switch(data)
+		if(60 to 100)	//Speed up after a minute
```

Comment:
@SonixApache Skully is thinking of system times which are in deciseconds (e.g.: spawn(60) is 6 seconds)